hack/setup.sh

```diff
@@ -109,6 +109,16 @@ function captureGitToken {
   fi
 }
 
+function captureGitlabToken {
```

Comment:
I couldn't find a function to capture the Gitlab host. How will the user configure the Gitlab host value?